examples/search_examples/public/search/app.tsx

```diff
@@ -390,6 +436,33 @@ export const SearchExamplesApp = ({
                   />
                 </EuiButtonEmpty>
               </EuiText>
+              <EuiSpacer />
```

Comment:
I think the structure of the page is very messy at this point.
It seems to me that this would fit better in the first (Searching Elasticsearch usng `data.search`) section.

What do you think about having an item there titled "Handling partial results" in that section?
Any other ideas?